### PR TITLE
ACP-2521 Added SalesPaymentDetail docs and update Stripe docs.

### PR DESCRIPTION
**Pull request by stereomon**
## PR Description

- Added SalesPaymentDetail module integration and usage.
- Updated Stripe guide with SalesPaymentDetail.

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/payment-details.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+title: Payment details of third party integrations
+description: Find out about how to use payment details from a third-party payment service provider in your Spryker shop.
```
Comment (matweew) [inline docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/payment-details.md:3]:
```suggestion
description: Find out about how to save payment details from a third-party payment service provider in your Spryker shop.
```

**Review by matweew — APPROVED** [review]

File: docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/payment-details.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        ];
+    }
+
+    //...
+}
+```
+
+## 5. Usage
+
+When payment is creating using a third-party payment service provider which supports this feature, you will receive asynchronous messages about the payment. When you want to use this data of the `spy_sales_payment_detail` table, you need to combine the data from this table with the entity you are fetching from the database where this payment detail is related to.
```
Comment (andriitserkovnyi) [inline docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/payment-details.md:96]: Make this a more clear message by rewriting/breaking down into more sentences

File: docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/payment-details.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
+---
+title: Retrieve and use payment details from third-party payment providers
+description: Learn how to retrieve and use payment details from a third-party payment service providers
+draft: true
```
Comment (andriitserkovnyi) [inline docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/payment-details.md:4]: Does this need to be published or just merged?

Comment (stereomon) [reply docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/payment-details.md:4]: Should be published as well

**Pull request merged**